backend/Terraform/modules/management_api/rest_api_getway.tf

```diff
@@ -0,0 +1,158 @@
+resource "aws_api_gateway_rest_api" "management_api" {
+  name        = var.api_name
+  endpoint_configuration {
+    types = ["REGIONAL"]
+  }
+}
+
+resource "aws_api_gateway_resource" "announcement_lambda" {
```

Comment:
make this resource run in a for_each statement so that lambda function paths are dynamic and not fixed